template/en/default/global/header.html.tmpl

```diff
@@ -131,7 +130,7 @@
     <title>[% title %]</title>
 
     [% IF Param('utf8') %]
-      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+      <meta charset="UTF-8">
```

Comment:
👍

Reply:
Happy New Years!!!
And anything else?

Reply:
I think that's it! Let's just wait for the tests to finish.